.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
-Issue #: https://github.com/bcgov/met-public/issues/
+Issue #: https://citz-gdx.atlassian.net/browse/DESENG-XXX
 
-*Description of changes:*
+**Description of changes:**
+- 
 
-
-By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).
+**User Guide update ticket (if applicable):**
```

Comment:
thanks for getting rid of that boilerplate! and I like the forward thinking with user guide ticket integration